CreeDictionary/constants/__init__.py

```diff
@@ -137,3 +137,10 @@ class Analysis(NamedTuple):
     raw_prefixes: str
     lemma: str
     raw_suffixes: str
+
```

Comment:
Does the following code in `utils/fst_analysis_parser.py` belong here?

https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/1e6479cc855f36706923d0c713a0bed0a6a912b3/CreeDictionary/utils/fst_analysis_parser.py#L59-L68

Reply:
Like an `unconcatenate()`? Yeah, that would be good!